docs/glossary.md

```diff
@@ -33,3 +33,10 @@ In the context of this extension and oCIS in general, we are using the following
 
 - Collection of related settings
 - Registered by an ocis extension
+
+### Settings Value
+
+- Manifestation of a setting for a specific user
+- E.g. used for customization (at runtime) in `ocis-web`
+- `ocis-web` extension for modifying settings values is provided by this service
```

Comment:
oh, I wasn't aware of this one. is `ocis-web` an extension on the frontend? or is it actually an ocis extension?

note: we should make the term extension less ambiguous, I identify 2 things:

1. ocis extension (i.e: `ocis-konnectd`, `ocis-proxy`...)
2. phoenix extension _point_?

Reply:
`ocis-web` = phoenix
`ocis-web-settings` = ocis-web extension or app. but we agreed to call it extension.

the term `extension points` is already used in ocis-web for e.g. the indicators extension points = possibility to inject additional components into a web ui.

What I meant here in the glossary, is, that ocis-settings delivers the settings.js as ocis-web extension, i.e. `ocis-web-settings`.

Reply:
Fixed the extension name

Reply:
updating my personal knowledge base with `ocis-web === phoenix`. Thank you!